components/terrain/chunkmanager.hpp

```diff
@@ -32,14 +32,17 @@ namespace Terrain
 
         osg::ref_ptr<osg::Node> getChunk(float size, const osg::Vec2f& center, int lod, unsigned int lodFlags);
 
+        void setCullingActive(bool active) { mCullingActive = active; }
+        void setCompositeMapSize(unsigned int size) { mCompositeMapSize = size; }
+        void setCompositeMapLevel(float level) { mCompositeMapLevel = level; }
+        void setMaxCompositeGeometrySize(float maxCompGeometrySize) { mMaxCompGeometrySize = maxCompGeometrySize; }
+
```

Comment:
I haven't seen function definitions doing something in headers in OpenMW yet, is this the preferred way for such short functions?

Reply:
Actually, we use approach (in the `apps/openmw/mwgui/formatting.hpp`), for example.
From what I can tell, there is no strict rule here.